apps/src/studio/ThreeSliceAudio.js

```diff
@@ -0,0 +1,127 @@
+/** @file A three-part loopable audio effect (start-loop-end). */
+/* jshint
+ funcscope: true,
+ newcap: true,
+ nonew: true,
+ shadow: false,
+ unused: true,
+ eqeqeq: true,
+
+ maxlen: 90,
+ maxstatements: 200
+ */
+'use strict';
+
+var debugLogging = false;
```

Comment:
You mean [like this](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/netsim/NetSimLogger.js)?  Yes, I would like us to move in this direction :grin: